Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN yum install -y git \
                    make \
                    sudo \
                    rh-nodejs8 \
+                   file \
```

Comment:
@duduribeiro not that it's a problem or anything, just wondering why this is needed?

Reply:
@gsaslis we need it since ImageMagick uses it to verify if a file is an image or not.

our ruby-24-centos image had it, but it was removed from ruby-25-centos.